README.md

```diff
@@ -1,2 +1,84 @@
-# charts
-Contane Helm Charts
+# Contane Helm Charts
+Welcome to the Contane Helm Charts repository! Here, you'll find a collection of Helm Charts for all the open-source projects developed by Contane. These charts are designed to simplify the deployment and management of applications on Kubernetes clusters.
+
+## Features
+- **Ready to Use**: All Helm Charts in this repository are pre-configured for immediate use. They can be deployed with default settings, making it easy to get started.
+- **Customizable**: For production environments, we highly recommend configuring the charts to suit your specific needs. Each chart provides various configuration options to tailor the deployment.
+- **Consistent Updates**: Our charts are regularly updated to ensure compatibility with the latest versions of Kubernetes and Helm, as well as to include new features and bug fixes.
+
+## Prerequisites
+Before you begin, ensure you have the following:
+
+- **A Running Kubernetes Cluster**: A functional Kubernetes cluster is required to deploy the Helm Charts. You can set up a cluster using Minikube, Kubernetes in Docker (Kind), or any cloud provider like GKE, EKS, or AKS.
+- **Helm 3.x Installed**: Ensure that you have Helm version 3.x installed on your local machine. You can follow the official Helm installation guide for detailed instructions.
+
+## Installation of Contane Helm Charts
+Installing a Contane Helm Chart is straightforward. Follow these steps to get started:
+
+### Using the Helm Chart Repository
+
+1. **Add the Repository**:
+    ```bash
+    helm repo add contane https://ghcr.io/contane/charts
+    helm repo update
+    ```
+
+2. **Install a Chart**:
+    Replace `<chart-name>` with the name of the chart:
+    ```bash
+    helm install my-custom-release contane/<chart-name>
+    ```
+
+3. **Verify the Installation**:
+    ```bash
+    helm list
+    kubectl get all -l app.kubernetes.io/instance=my-custom-release
+    ```
+
+### Using the OCI Registry
+
+1. **Install a Chart**:
+    Replace `<chart-name>` with the name of the chart:
+    ```bash
+    helm install my-custom-release oci://ghcr.io/contane/charts/<chart-name>
+    ```
+
+2. **Verify the Installation**:
+    ```bash
+    helm list
+    kubectl get all -l app.kubernetes.io/instance=my-custom-release
+    ```
+
+## Configuration
+While the charts come with sensible defaults, you can customize their behavior by providing your own values. To do this, you can create a `values.yaml` file and pass it during installation:
```

Comment:
At the start, we are recommending to always customize production deployments, so the tone of this section should reflect that.

```suggestion
You can customize the charts by creating a `values.yaml` file and providing it during installation:
```